### ISSUE TITLE
ps: retry on `plustekctl` crash

### ISSUE THREAD

**Issue by eventualbuddha** [votingworks/vxsuite#958]

**Comment by eventualbuddha** [comment]
So far I'm seeing `plustekctl` being unable to run again after being `kill`ed:

```
~/src/plustekctl main brian@vx
❯ make demo
cat config.example.json | sed 's|$PWD|/home/brian/src/plustekctl|g' > config.json
mkdir -p files
./plustekctl --config config.json --delimiter ''
Reading config from config.json
Getting ready to scan…
Getting scanner devices…
Finding active scanner…
Opening scanner (device=pl99102:libusb:001:003, scanner=Plustek 776U scanner)…
[SendP99102ExeCmd:218] === Got error 9 trying to write, wrote: 0. ===
[SendP99102Cmd] i(0) ret error code: 9
[SendP99102ExeCmd:218] === Got error 9 trying to write, wrote: 0. ===
[SendP99102Cmd] i(1) ret error code: 9
[SendP99102ExeCmd:218] === Got error 9 trying to write, wrote: 0. ===
[SendP99102Cmd] i(2) ret error code: 9
error: failed to open scanner (device=pl99102:libusb:001:003, scanner=Plustek 776U scanner)
Makefile:28: recipe for target 'demo' failed
make: *** [demo] Error 1
```

**Comment by eventualbuddha** [comment]
Disconnecting and reconnecting the scanner makes that issue go away, but that's not really something we can do in production.

**Comment by eventualbuddha** [comment]
Fixed by #971
